tests/onnx/quantization/test_ptq_params.py

```diff
@@ -87,7 +88,7 @@ def test_quantize_outputs(quantize_outputs):
     min_max_algo._backend_entity = ONNXMinMaxAlgoBackend()
     nncf_graph = NNCFGraphToTest().nncf_graph
     assert min_max_algo._parameters.quantize_outputs == quantize_outputs
-    q_setup = min_max_algo._get_quantizer_setup(nncf_graph)
+    q_setup = min_max_algo._get_quantizer_setup(nncf_graph, HWFusedPatterns().get_full_pattern_graph())
```

Comment:
```suggestion
    q_setup = min_max_algo._get_quantizer_setup(nncf_graph, GraphPattern())
```
Here and below if the change is applicable.

Reply:
Added. Thanks!